src/kernel-install/uki.conf

```diff
@@ -0,0 +1,34 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it under the
+#  terms of the GNU Lesser General Public License as published by the Free
+#  Software Foundation; either version 2.1 of the License, or (at your option)
+#  any later version.
+#
+# Ini-style configuration file for ukify(1) which is only effective when
+# $KERNEL_INSTALL_LAYOUT or layout= in install.conf is set to uki and
+# $KERNEL_INSTALL_UKI_GENERATOR or uki_generator= in install.conf is set to
+# ukify, or is unset. $KERNEL_INSTALL_CONF_ROOT may be used to override the 
+# search path.
+#
+# See kernel-install(8) for details.
+
+#[UKI]
+#Initrd=
+#Microcode=
+#Splash=
+#PCRPKey=
+#PCRBanks=
+#SecureBootSigningTool=
+#SecureBootPrivateKey=/etc/kernel/secure-boot-key.pem
+#SecureBootCertificate=/etc/kernel/secure-boot-cert.pem
+#SecureBootCertificateDir=
+#SecureBootCertificateName=
+#SecureBootCertificateValidity=
+#SigningEngine=
+#SignKernel=
+
+#[PCRSignature:NAME]
+#PCRPrivateKey=/etc/systemd/tpm2-pcr-key.pem
```

Comment:
hmm, the systemd-measure man page calls this file `tpm2-pcr-private.pem`. I think we should stick to that.